robottelo/cli/base.py

```diff
@@ -1,28 +1,27 @@
 # -*- encoding: utf-8 -*-
 """Generic base class for cli hammer commands."""
 import logging
-import re
 
 from robottelo import ssh
 from robottelo.cli import hammer
 from robottelo.config import conf
 
 
-# Task status message have two formats:
-#   * "Task b18d3363-f4b8-44eb-871c-760e51444d22 success: 1.0/1, 100%,
-#     elapsed: 00:00:02\n"
-#   * "Task 6ba9d82a-ea55-4934-8aab-0e057102ee11 running: 0.005/1, 0%, 0.0/s,
-#     elapsed: 00:00:02, ETA: 00:06:55\n"
-TASK_STATUS_REGEX = re.compile(
-    r'Task [\w-]+ \w+: [\d./]+, \d+%,( [\d./s]+,)? elapsed: [\d:]+'
-    '(, ETA: [\d:]+)?\n\n?'
-)
-
-
 class CLIError(Exception):
     """Indicates that a CLI command could not be run."""
 
 
+class CLICommandExecutionError(Exception):
+    """Indicates that a CLI command returned an error after execution."""
+    def __init__(self, return_code, stderr, msg):
+        self.return_code = return_code
+        self.stderr = stderr
+        self.msg = msg
+
+    def __str__(self):
+        return self.msg
```

Comment:
And there's such need - we don't just need some new exception, we need an exception, which will contain `return_code` and `stderr` in it, as we have some tests, which are expecting some specific `return_code` value.
E.g., defining `self.return_code` in `__init__` section will allow us to check for a custom `return_code` like this:
```python
with self.assertRaises(CLICommandExecutionError) as e:
    ContentView.info({'id': 123})
self.assertEqual(e.exception.error_code, 1)
```